app/views/streamrules/index.scala.html

```diff
@@ -16,76 +18,7 @@
-                </p>
-
-                <form class="form-inline manual-selector-form" style="margin-bottom: 10px; margin-top: 10px;" id="message-selector">
-
-                    <div class="form-group">
-                        <input type="text" class="form-control" name="message_id" placeholder="Message ID" required="true" style="width: 300px;">
-                    </div>
-
-                    <div class="form-group">
-                        <input type="text" name="index" class="form-control" placeholder="Index" required="true">
-                    </div>
-
-
-                    <button type="submit" class="btn btn-info manual-selector-load" data-stream-id="@stream.getId">
-                        <i class="fa fa-magic"></i> Load message
-                    </button>
-                </form>
-
-                The <em>message ID</em> and <em>index</em> can be found in the detail view of any message.
-            </div>
-            <div class="spinner" style="display: none;"><h2><i class='fa fa-spinner fa-spin'></i> &nbsp;Loading message</h2></div>
-
-            <div class="sample-message-display" style="display: none; margin-top: 5px;">
-                <strong>Next step:</strong>
-                Add/delete/modify stream rules in step 2 and see if the example message would have been routed into the stream or not.
-
-                <div class="alert alert-info" id="sample-message" style="margin-top: 10px; margin-bottom: 0px;">
-                    <h3>Message {{messageId}}<span></span></h3>
-
-                    <dl class="sample-message-fields">
-                        <dt data-field="{{field}}" data-value="{{raw-value}}" data-occurrence="repeat">{{field}}</dt><dd data-occurrence="repeat">{{value}}</dd>
-                    </dl>
-                </div>
-            </div>
-
-            <hr />
-
-            <div class="buttons pull-right">
-                <button class="btn btn-success btn-sm show-stream-rule" data-stream-id="@stream.getId">
-                    Add stream rule
-                </button>
-            </div>
-
-            <h2>
-                <i class="fa fa-cogs"></i>
-                2. Manage stream rules
-            </h2>
-
-            The example message would have been routed into this stream if every rule below has a green background.
-
-            <div class="streamrules-list-container streamrule-checker" data-stream-id="@stream.getId">
-                @views.html.partials.streamrules.list_well.render(streamrules)
-                @views.html.partials.streamrules.stream_rule_form.render(stream.getId, null)
-            </div>
-
-            <p style="margin-top: 10px;">
-                <a href="@routes.StreamsController.index()" class="btn btn-success">I'm done!</a>
-            </p>
-        </div>
+    <div class="react-streamrules-editor" data-stream-id="@stream.getId" data-permissions='@Json.toJson(currentUser.getPermissions)'>
     </div>
 
     <script type="text/javascript">
```

Comment:
I think this inline script can be deleted now, as you do the same here: https://github.com/Graylog2/graylog2-web-interface/pull/1264/files#diff-c22136bc9bccf7b583b30d3a8829c736R11